apps/cookbook/src/app/shared/api-description/api-description-methods/api-description-methods.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from '@angular/core';
+
+export interface ApiDescriptionMethod {
+  name: string;
+  description?: string;
+  signature?: string;
+}
+
+export interface ApiDescriptionMethodColumns {
+  Name: string;
+  Description?: string;
+  Type?: string;
```

Comment:
We use camel-casing:
```suggestion
  name: string;
  description?: string;
  type?: string;
```